$MirOS: ports/plan9/kencc/patches/patch-utils_ka_a_h,v 1.1 2006/03/31 03:50:20 tg Exp $
--- utils/ka/a.h.orig	Thu Mar 30 23:54:34 2006
+++ utils/ka/a.h	Thu Mar 30 23:55:24 2006
@@ -120,7 +120,7 @@ EXTERN	Biobuf	obuf;
 
 void*	alloc(long);
 void*	allocn(void*, long, long);
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 void	pushio(void);
 void	newio(void);
 void	newfile(char*, int);
